index.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const mongoose = require("mongoose")
+const app = express();
+var cors = require('cors')
+
+const VideoLibraryRouter = require("./Routes/VideoLibrary.route")
+const { addToDatabase } = require("./Database/addToDatabase.js")
+const { databaseConnection } = require("./Database/databaseConnection.connect.js")
+const PORT = 3000;
+
+app.use(cors());
```

Comment:
Here it is fine,
But in the future just to add a layer of security and also just to keep the API for yourself, you could make use of the cors the restrict the API access to your domain.

something like this if you haven't tried out already:

```
app.use(cors({
    origin: "Your domain"
}))
```

https://expressjs.com/en/resources/middleware/cors.html

Reply:
I have fixed this issue.

Reply:
I have fixed this issue.